crates/symbolicator/src/sources.rs

```diff
@@ -125,38 +127,59 @@ pub struct FilesystemSourceConfig {
 }
 
 /// Local helper to deserialize an S3 region string in `S3SourceKey`.
-fn deserialize_region<'de, D>(deserializer: D) -> Result<rusoto_core::Region, D::Error>
+fn deserialize_region<'de, D>(deserializer: D) -> Result<Region, D::Error>
 where
     D: Deserializer<'de>,
 {
     // This is a Visitor that forwards string types to rusoto_core::Region's
     // `FromStr` impl and forwards tuples to rusoto_core::Region's `Deserialize`
     // impl.
-    struct RusotoRegion;
+    struct SdkRegion;
 
-    impl<'de> serde::de::Visitor<'de> for RusotoRegion {
-        type Value = rusoto_core::Region;
+    impl<'de> serde::de::Visitor<'de> for SdkRegion {
+        type Value = Region;
 
         fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
-            formatter.write_str("string or tuple")
+            formatter.write_str("string")
```

Comment:
is this type change going to be a problem for some configurations?

Reply:
Yes; custom regions, which use tuples per the [docs](https://github.com/getsentry/symbolicator/blob/6fd6e55fb0fd6188638bcbf6d5697fafe9d1090e/docs/api/index.md), don't currently work, as the AWS Rust SDK doesn't support them.

Reply:
I don't think this is in-scope for the AWS SDK, at any rate it doesn't support it today so if we want to offer this functionality we'll have to add it ourselves. :-/